README.md

```diff
@@ -1,210 +1,305 @@
+— where all programming symbols are arranged for comfort and efficiency — and
+emits <kbd>Return</kbd> when tapped;
+- the spacebar brings the <kbd>Navigation</kbd> layer when held (in orange).
 
-![alt, navigation and sym layers under the thumbs](img/base_easy.svg)
+![alt, navigation and sym layers under the thumbs](./img/layer_taps.svg)
 
 Having <kbd>Backspace</kbd> and <kbd>Enter</kbd> under the thumbs is enough to
-reduce the pinky fatigue very significantly. And using the <kbd>Symbol</kbd>
-and <kbd>Navigation</kbd> layer further reduces hand and finger movements.
+reduce the pinky fatigue very significantly. And using the <kbd>Symbols</kbd>
+and <kbd>Navigation</kbd> layers further reduces hand and finger movements.
+
 
-### 2. Enable the Home-Row Mods
+#### Next level: enable the Home Row Mods
 
-When you are familiar with mod-taps, it’s time to enable them on the homerow
-with the “hrm” variants:
+When you are familiar with mod-taps, it’s time to enable them on the home row
+with the “HRM” variants:
 
 - <kbd>FDS</kbd> and <kbd>JKL</kbd> become <kbd>Ctrl</kbd>, <kbd>Alt</kbd>,
 <kbd>Super</kbd> when held long enough;
 - the left thumb key can now emit a <kbd>Shift</kbd> rather than <kbd>Alt</kbd>
 when held.
 
-![homerow mods on SDF keys](img/base_hrm.svg)
+![home row mods on SDF keys](./img/hrm.svg)
+
+This is a very basic variant of the [Miryoku] principle: one layer on each
+thumb key, and symmetrical modifiers on the home row.
+
+
+### 3. Symbols layer
 
-This is a very basic variant of the [Miryoku][1] principle: one layer on each
-thumb key, and symmetrical modifiers on the homerow.
+For the <kbd>Symbols</kbd> layer you can keep <kbd>AltGr</kbd> as-is. It is
+useful for keyboard layouts that rely heavily on the <kbd>AltGr</kbd> key.
 
-### 3. Spice It Up
+But the real fun (especially for programmers) happens when we enable the “One 
+Dead Key” (= 1DK) programmation layer!
 
-- the 300 ms delay before a key becomes a modifier has been chosen to be easy
-for beginners. Once used to mod-taps, you may want to reduce it so keyboard
-shortcuts can be done more quickly;
-- more layers available to enable (Vim-navigation, num-row, etc).
+![1dk symbols layer on a 33-key keyboard](./img/symbols.svg)
 
-#### A Vim-friendly mod:
 
-- 3 home-row mods per hand for <kbd>Ctrl</kbd>, <kbd>Alt</kbd>, <kbd>Super</kbd>
-- 3 layer-tap keys under the thumbs: <kbd>Shift</kbd>/<kbd>Backspace</kbd>,
-<kbd>Navigation</kbd>/<kbd>Space</kbd>, <kbd>Symbol</kbd>/<kbd>Return</kbd>
+#### Num row >> Num pad
+
+If enabled, in <kbd>Symbols</kbd> mode, pressing the left thumb key brings up
+the <kbd>NumRow</kbd> layer:
+
+- all digits are on the home row, in the order you already know
+- the upper row helps with <kbd>Shift</kbd>-digit shortcuts
+- the lower row has dash, comma, dot and slash signs to help with number / date
+inputs
+- <kbd>Space</kbd> becomes a narrow no-break space for layouts that supports it
```

Comment:
```suggestion
- <kbd>Space</kbd> becomes <kbd>Shift</kbd><kbd>Space</kbd>, which can be
a no-break space for layouts that support it
```

Reply:
It happens to be shift+space because the layout we support for now have their narrow no-break space with this combo, but the idea is mostly to produce a nnbs, not necessarily with this combo if the defalias of the keyboard layout define nnbs with another combo (I hope you get what I mean by that).